bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/Filters.java

```diff
@@ -0,0 +1,499 @@
+     */
+    public Filter regex(String regex) {
+      return new SimpleFilter(RowFilter.newBuilder().setFamilyNameRegexFilter(regex).build());
+    }
+
+    /** Matches only cells from columns whose families match the value. */
+    public Filter exactMatch(String value) {
+      return new SimpleFilter(
+          RowFilter.newBuilder().setFamilyNameRegexFilter(RegexUtil.literalRegex(value)).build());
+    }
+  }
+
+  public static class QualifierFilter {
+
+    /**
+     * Matches only cells from columns whose qualifiers satisfy the given <a
+     * href="https://github.com/google/re2/wiki/Syntax>RE2 regex</a>. Note that, since column
+     * qualifiers can contain arbitrary bytes, the `\C` escape sequence must be used if a true
+     * wildcard is desired. The `.` character will not match the new line character `\n`, which may
+     * be present in a binary qualifier.
+     */
+    public Filter regex(String regex) {
+      return new SimpleFilter(
+          RowFilter.newBuilder()
+              .setColumnQualifierRegexFilter(ByteString.copyFromUtf8(regex))
+              .build());
+    }
+
+    /** Matches only cells from columns whose qualifiers equal the value. */
+    public Filter exactMatch(ByteString value) {
+      return new SimpleFilter(
+          RowFilter.newBuilder()
+              .setColumnQualifierRegexFilter(RegexUtil.literalRegex(value))
+              .build());
+    }
+
+    /**
+     * Matches only cells from columns within the given range.
+     */
+    public static class RangeBuilder {
+      private ColumnRange.Builder range = ColumnRange.newBuilder();
+
+      public RangeBuilder family(String family) {
+        range.setFamilyName(family);
+        return this;
+      }
+
+      public RangeBuilder startClosed(ByteString value) {
+        range.setStartQualifierClosed(value);
+        return this;
+      }
+
+      public RangeBuilder startOpen(ByteString value) {
+        range.setStartQualifierOpen(value);
+        return this;
+      }
+
+      public RangeBuilder endClosed(ByteString value) {
+        range.setEndQualifierClosed(value);
+        return this;
+      }
+
+      public RangeBuilder endOpen(ByteString value) {
+        range.setEndQualifierOpen(value);
+        return this;
+      }
+
+      public Filter build() {
+        return new SimpleFilter(
+          RowFilter.newBuilder().setColumnRangeFilter(range.build()).build());
+      }
+    }
+
+    /**
+     * Construct a {@link RangeBuilder} that can create a {@link ColumnRange} oriented
+     * {@link Filter}.
+     * @return a new {@link RangeBuilder}
+     */
+    public RangeBuilder range() {
+      return new RangeBuilder();
+    }
+  }
+
+  public static class TimestampFilter {
+    /**
+     * Matches only cells with timestamps within the given range.
+     *
+     * @param start Inclusive start of the range.
+     * @param end Exclusive end of the range.
+     */
+    public Filter range(Long start, Long end) {
```

Comment:
This should follow the same pattern as the range above, but exclude the openStart & closedEnd methods